src/components/single-line-diagram-viewer/single-line-diagram-viewer.ts

```diff
@@ -120,7 +120,7 @@ export class SingleLineDiagramViewer {
     arrowSvg: string;
     arrowHoverSvg: string;
     svgDraw: Svg | undefined;
-    handleTogglePopover: HandleTogglePopoverType | null;
+    handleTogglePopover: OnToggleSLDHoverCallbackType | null;
```

Comment:
you'd better change the variable name also. And acronym should not have full upper case in variable/class names
```suggestion
    onToggleHoverCallback: OnToggleSldHoverCallbackType | null;
```

Reply:
You forgot to change the variable name, I'll take care of it.